management_api_app/api/routes/workspaces.py

```diff
@@ -23,12 +24,22 @@ async def retrieve_active_workspaces(workspace_repo: WorkspaceRepository = Depen
 
 @router.post("/workspaces", status_code=status.HTTP_202_ACCEPTED, response_model=ResourceInResponse, name=strings.API_CREATE_WORKSPACE)
 async def create_workspace(workspace_create: ResourceInCreate, workspace_repo: WorkspaceRepository = Depends(get_repository(WorkspaceRepository))) -> ResourceInResponse:
```

Comment:
instead of creating service bus here could you not also use Depends like we have for repository?

Reply:
I have an opinion about these types of dependency management features. I will not disclose it here.

In its current state I don't see any benefit using it for ServiceBus class.